Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -391,7 +417,8 @@ void Arm64FPRCache::Flush(FlushMode mode, PPCAnalyst::CodeOp* op)
   {
     const RegType reg_type = m_guest_registers[i].GetType();
 
-    if (reg_type != RegType::NotLoaded && reg_type != RegType::Immediate)
+    if (reg_type != RegType::NotLoaded && reg_type != RegType::Discarded &&
+        reg_type != RegType::Immediate)
```

Comment:
I assume FlushRegister shall assert if the reg_type is Discarded, so we should call it here and assert there. This will tell us if there is a bug here in this code.
However this will also trigger on interpreter fallbacks, so I'm not sure about this signaling...

anyways, the next comment is very outdated.

Reply:
With interpreter fallbacks in mind, I think I would like to keep it the way it is.

> anyways, the next comment is very outdated.

What are you referring to by this?

Reply:
I was talking about `XXX: Determine if we can keep a register in the lower 64bits \n Which will allow it to be callee saved.` However I don't care if you drop it or not.